tests/components/owntracks/test_helper.py

```diff
@@ -0,0 +1,30 @@
+"""Test the owntracks_http platform."""
+from unittest.mock import patch
+import pytest
+
+from homeassistant.components.owntracks import helper
+
+
+@pytest.fixture(name="nacl_imported")
+def mock_nacl_imported():
+    """Mock a successful import."""
+    with patch("homeassistant.components.owntracks.helper.nacl"):
+        yield
+
+
+@pytest.fixture(name="nacl_not_imported")
+def mock_nacl_not_imported():
+    """Mock non successful import."""
+    with patch("homeassistant.components.owntracks.helper.nacl"):
```

Comment:
I think this is better. But it will still not make the ImportError code path covered.
```suggestion
    with patch("homeassistant.components.owntracks.helper.nacl", new=None):
```